### ISSUE TITLE
[Bug]: Can not import any highlighting

### ISSUE THREAD

**Issue by its-felix** [cloudscape-design/code-view#14]
### Browser

Chrome

### Package version

3.0.4

### React version

18.2.0

### Description

I can not get the importing to work like in the examples.
I have tried both:
```ts
import jsonHighlight from '@cloudscape-design/code-view/highlight/json'; // like in the examples
// and
import jsonHighlight from '@cloudscape-design/code-view/code-view/highlight/json'; // suggested by my IDE
```

For the first variant, I see the following error:
```
 error TS2307: Cannot find module '@cloudscape-design/code-view/highlight/json' or its corresponding type declarations.

2 import jsonHighlight from '@cloudscape-design/code-view/highlight/json';
                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

For the second variant, this error:
```
[commonjs--resolver] Missing "./code-view/highlight/json" specifier in "@cloudscape-design/code-view" package
error during build:
Error: Missing "./code-view/highlight/json" specifier in "@cloudscape-design/code-view" package
```

### Source code

```ts
import jsonHighlight from '@cloudscape-design/code-view/highlight/json'; // like in the examples
// and
import jsonHighlight from '@cloudscape-design/code-view/code-view/highlight/json'; // suggested by my IDE
```

### Reproduction

1. Try to import any highlighting
2. build
3. fail

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/cloudscape-design/components/blob/main/CODE_OF_CONDUCT.md)
- [X] I checked the current issues on [this](https://github.com/cloudscape-design/code-view/issues) and [main](https://github.com/cloudscape-design/components/issues) repositories for duplicate problems

**Comment by just-boris** [comment]
You need to set `"moduleResolution": "bundler"` in your tsconfig.

More details in their docs: https://www.typescriptlang.org/docs/handbook/modules/reference.html#bundler

**Comment by hunoz** [comment]
I know this has been closed, but I encountered the same issue and implemented the fix suggested by @just-boris, but this lead to issues with the regular @cloudscape-design/components package:
```
import { OptionDefinition } from '@cloudscape-design/components/internal/components/option/interfaces';
```

now throws an error:
```
Cannot find module '@cloudscape-design/components/internal/components/option/interfaces' or its corresponding type declarations.
```

This happens with pretty much every interface, and my VSCode recommends changing the import path to  `node_modules/@cloudscape-design/components/internal/components/option/interfaces`, which doesn't seem right.

Is this intentional? Should we be importing the packages like that instead?
